src/common/Pagination/Pagination.tsx

```diff
@@ -13,6 +13,7 @@ interface PaginationProps {
   pageSize: number;
   totalPages: number;
   pageSizeOptions?: [];
+  paginated: boolean;
```

Comment:
this wording seems odd to parameterize the Pagination component with a `paginated` property. The definition of the component is that it is `paginated` always, but in this case we are changing the ability to select the `pageSize` not enabling/disabling the entire component. Maybe we change `paginated` to something like `pageSizeSelectable` as this is the only affected portion of the component.

Reply:
it makes sense!